### PR TITLE
test(e2e): scroll bounce test added

### PR DESCRIPTION
**Pull request by oscard0m**

### CODE REVIEW

File: experiences/homepage/homepage.test.js [hunk 1]
```diff
@@ -47,6 +47,66 @@ const homepageTest = function() {
...
+		);
+
+		expect(isAtExpectedSnapPoint).equal(true);
+
+		await scrollBy(browser, 20);
+
+		isAtExpectedSnapPoint = await isAtSnapPoint(
+			browser,
+			config.cards.homepage.cardSelector,
+			config.cards.homepage.snapPoints.minimised - 0.1
```
Comment (oscard0m) [inline experiences/homepage/homepage.test.js:75]: I don't like this magic number. We should calculate it as we transform percentage with `window.innerHeight`, etc. but this should be something out of the scope in my opinion.

File: experiences/homepage/homepage.test.js [hunk 2]
```diff
@@ -47,6 +47,66 @@ const homepageTest = function() {
...
+		);
+		expect(isAtExpectedSnapPoint).equal(true);
+
+		await scrollBy(browser, 20);
+
+		isAtExpectedSnapPoint = await isAtSnapPoint(
+			browser,
+			config.cards.homepage.cardSelector,
+			config.cards.homepage.snapPoints.minimised - 0.2
```
Comment (oscard0m) [inline experiences/homepage/homepage.test.js:84]: same as here: https://github.com/Marfeel/flowcards-playground/pull/161/files#r621455190

Comment (oscard0m) [inline experiences/homepage/homepage.test.js:84]: I did not wrapped this inside a loop due to the problem with loops and async/await statements. Any idea on how to write with less code repetition?

File: experiences/homepage/homepage.test.js [hunk 3]
```diff
@@ -47,6 +47,66 @@ const homepageTest = function() {
...
+
+		// small scroll up bounds
+		await scrollBy(browser, -20);
+		isAtExpectedSnapPoint = await isAtSnapPoint(
+			browser,
+			config.cards.homepage.cardSelector,
+			config.cards.homepage.snapPoints.minimised
+		);
+		expect(isAtExpectedSnapPoint).equal(true);
+	});
```
Comment (oscard0m) [inline experiences/homepage/homepage.test.js:108]: Same as previous comments applies here:
- Magic numbers
- Code repetition

File: experiences/homepage/homepage.test.js (hunk 1 above)
Comment (davidmohringmrf) [reply experiences/homepage/homepage.test.js:75]: yep, also scrolling down 20 may not be 0.1 always, right? as the height of the screen can change depending on the devices?

File: experiences/homepage/homepage.test.js (hunk 2 above)
Comment (davidmohringmrf) [reply experiences/homepage/homepage.test.js:84]: `await arrayOfScrollsToDo.reduce((prev, value)=>{
		return prev.then(()=>{
			scrollBy(browser, value);
			checkSnappoint();
			return new Promise(resolve => setTimeout(resolve, 100));
		});
	}, Promise.resolve());`
	
	Something like this

Comment (davidmohringmrf) [reply experiences/homepage/homepage.test.js:84]:
```
await arrayOfScrollsToDo.reduce((prev, value)=>{
		return prev.then(async()=>{
			scrollBy(browser, value);
			return await checkSnappoint();
		});
	}, Promise.resolve());
```

Comment (davidmohringmrf) [reply experiences/homepage/homepage.test.js:84]: something like this could work, chaining the checks

**Comment by oscard0m** [comment]
Implemented in https://github.com/Marfeel/flowcards-playground/pull/166/

**Pull request closed without merging**